packages/react-theme/src/utils/mergeFallbackCssVariable.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Function that accepts either a var(--value) or --value CSS prop, and returns a var()
+ * declaration with a fallback: var(--value, fallback).
+ * If an undefined or non-string cssVariable is passed in, the fallback is returned directly.
+ */
+export function mergeFallbackCssVariable(
+  cssVariable: string | undefined,
+  fallback: string | undefined,
```

Comment:
Wouldn't you want both of these to always be defined?

Reply:
the original `cssVariable` may not be defined in the high contrast use case, since ideally teams won't be defining high contrast variables. I could make fallback required, though! I think I would then take out the handling of a non-existent fallback, which would also have the benefit that the fn would be typed to always return a string.